control_panel_api/tests/test_aws.py

```diff
@@ -4,7 +4,7 @@
 from control_panel_api import aws
 
 
-@patch('boto3.client')
+@patch('control_panel_api.aws.aws_api_client')
```

Comment:
Do you need to patch the `aws_api_client` if it's set to `MagicMock()`?

Reply:
Yes good shout, can just `import` it